Data/ByteString.hs

```diff
@@ -889,6 +891,22 @@ take n ps@(BS x l)
     | otherwise = BS x n
 {-# INLINE take #-}
 
+-- | /O(1)/ 'takeEnd' @n xs@ is equivalent to @'drop' ('length' xs - n) xs@.
+-- Takes @n@ elements from end of bytestring.
+--
+-- >>> takeEnd 3 $ pack (c2w <$> "abcdefg")
```

Comment:
`c2w` is an internal function. I don't think we should use it in the documentation here. Why not rely on the `IsString` instance or alternatively on the `IsList` instance?!

```suggestion
-- >>> takeEnd 3 "abcdefg"
```

Reply:
Could you fix the other examples in the same way?

Reply:
ah yes sorry, I was just working on the cleaning up `./test` totally missed this. Will make amend.